models/cart-discount/src/cart-discount-draft/presets/sample-data-fashion/summer-flips.spec.ts

```diff
@@ -0,0 +1,104 @@
+import { TCartDiscountDraft, TCartDiscountDraftGraphql } from '../../../types';
+import summerFlips from './summer-flips';
+
+describe('with the preset `summerFlips`', () => {
+  it('should return a cart discount draft', () => {
+    const cartDiscountDraft = summerFlips().build<TCartDiscountDraft>();
+
+    expect(cartDiscountDraft.value).toMatchInlineSnapshot(`
+      {
+        "distributionChannel": undefined,
+        "product": {
+          "id": undefined,
+          "key": "summer_dress",
+          "typeId": "product",
+        },
+        "supplyChannel": undefined,
+        "type": "giftLineItem",
+        "variantId": 1,
+      }
+    `);
```

Comment:
Now, the product `reference` has a stable `undefined` `id`, and other fields we don't care about like `distributionChannel` and `supplyChannel` are `undefined` as well